vertical-pod-autoscaler/pkg/admission-controller/config.go

```diff
@@ -68,8 +71,10 @@ func configTLS(cfg certsConfig, minTlsVersion, ciphers string, stop <-chan struc
 	}
 	if *cfg.reload {
 		cr := certReloader{
-			tlsCertPath: *cfg.tlsCertFile,
-			tlsKeyPath:  *cfg.tlsPrivateKey,
+			tlsCertPath:  *cfg.tlsCertFile,
+			tlsKeyPath:   *cfg.tlsPrivateKey,
+			clientCaPath: *cfg.clientCaFile,
+			mwc:          mutatingWebhookClient,
```

Comment:
I'm not too sure on the VPA's style guide, but what about using a full name here to match the other names?
```suggestion
			mutatingWebhookClient: mutatingWebhookClient,
```